.github/workflows/build-and-test-web-demo.yml

```diff
@@ -25,8 +28,14 @@ jobs:
       - uses: mymindstorm/setup-emsdk@v6
         with:
           version: 2.0.4
-      - name: check out
-        uses: actions/checkout@v1
+      - name: checkout (push)
+        if: ${{ github.event_name == 'push' }}
+        uses: actions/checkout@v2
+      - name: checkout (pull request)
```

Comment:
Can we give these steps the same name ("check out"), or do they need to be distinct?